
<launch>
	<param name="topic_pub" value="/cmd_vel" />
	<node name="hace" pkg="hace_core" type="hace_core"  output="screen" >
	    <param name="models" value="/vol/meka/nightly/share/openpose/models/" />
	    <param name="rgb_topic" value="/xtion/rgb/image_raw" />
	    <param name="rgb_info_topic" value="/xtion/rgb/camera_info" />
	    <param name="depth_topic" value="/xtion/depth_registered/image" />
	</node>
</launch>
